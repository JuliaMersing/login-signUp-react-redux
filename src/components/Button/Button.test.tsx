import React from 'react';
import { render, screen } from '@testing-library/react';
import { Button } from './Button';

describe('Button component', () => {
	test('should match snapshot', () => {
		render(<Button>Login</Button>);
		screen.getByRole('button', { name: /Login/i });
		expect('button').toMatchSnapshot();
	});
});
