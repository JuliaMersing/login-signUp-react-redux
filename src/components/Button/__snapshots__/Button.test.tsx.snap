// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should match snapshot 1`] = `"button"`;
